File: Dataset/Leetcode/test/94/244.java
  public List<Integer> XXX(TreeNode root) {
        List<Integer> result = new ArrayList<>();
        handle(root,result);
        return result;
    }

    void handle(TreeNode root, List<Integer> list){
        if (root!=null) {
            handle(root.left,list);
            list.add(root.val);
            handle(root.right,list);
        }
    }

File: Dataset/Leetcode/valid/100/244.js
 var XXX = function(p, q) {
    if(p&&q){
       return p.val==q.val && XXX(p.right,q.right) && XXX(p.left,q.left)
    }else if(!p&& !q){
        return true
    }else{
        return false
    }
};

